README.md

```diff
@@ -119,20 +138,53 @@ PunktF searches for the source path in the following order:
 
 ## Template Format
 
-### Replacement
+### Comments
+
+Comments can be inserted with `{{!-- ... --}}`. They will be ignored by the template
+parser and will not be transferred to the output.
+
+Example:
+
+```python
```

Comment:
Use `Handlebars` as language, which seems to be supported: https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L2375
but please test it first 😬

Reply:
Done